File: Component/BusiLib/src/main/java-person/model/RelationNumModel.java
package model;

import java.io.Serializable;

public class RelationNumModel implements Serializable {
    /**
     * relation : 1
     * cnt : 4
     */

    private int relation;
    private int cnt;

    public int getRelation() {
        return relation;
    }

    public void setRelation(int relation) {
        this.relation = relation;
    }

    public int getCnt() {
        return cnt;
    }

    public void setCnt(int cnt) {
        this.cnt = cnt;
    }

}

File: Component/BusiLib/src/main/java-person/com/component/person/model/RelationNumModel.kt
package com.component.person.model

import com.alibaba.fastjson.annotation.JSONField
import java.io.Serializable

class RelationNumModel : Serializable {
    /**
     * relation : 1
     * cnt : 4
     */

    @JSONField(name = "relation")
    var relation: Int = 0
    @JSONField(name = "cnt")
    var cnt: Int = 0
}
